src/addons/model.js

```diff
@@ -1,19 +1,20 @@
 /**
  * @fileoverview A general modelling class used by the Presenter's
  * getModel function.
+ * @flow
  */
 
-import { set, Emitter } from '../microcosm'
+import { type Microcosm, Emitter, set } from '../microcosm'
 
-function isObservable(binding) {
+function isObservable(binding: *): boolean {
   return binding && typeof binding.subscribe === 'function'
 }
 
-function isCallable(binding) {
+function isCallable(binding: *): boolean {
```

Comment:
Huh. It does, but I can only do it for functions. If I attempt to do it for a class method, it yells at me. 

My preference is to be explicit. "I meant for this, I didn't forget". What do you think?